sky/serve/core.py

```diff
@@ -150,11 +137,12 @@ def up(
             serve_utils.generate_remote_config_yaml_file_name(service_name))
         controller_log_file = (
             serve_utils.generate_remote_controller_log_file_name(service_name))
-        controller_resources_in_config = (
-            controller_utils.get_controller_resources(
-                controller_type='serve',
-                controller_resources_config=serve_constants.CONTROLLER_RESOURCES
-            ))
+        controller_resources = controller_utils.get_controller_resources(
+            controller_type='serve',
+            controller_name=controller_name,
+            controller_resources_config=serve_constants.CONTROLLER_RESOURCES,
```

Comment:
Similarly, should this `controller_resources_config` be inferred by the `controller_type` in the `controller_utils`?